lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -222,6 +222,23 @@ def _test_run_with_short_error_msg(self, task_class):
 class TestRescoreInstructorTask(TestInstructorTasks):
     """Tests problem-rescoring instructor task."""
 
+    def assert_task_output(self, output, **expected_output):
+        """
+        Check & compare output of the task
+        """
+        self.assertEquals(output.get('total'), expected_output.get('total'))
```

Comment:
`self.assertEquals` has been deprecated since Python 2.7 use `self.assertEqual` instead: https://docs.python.org/2/library/unittest.html#deprecated-aliases